### PR TITLE
Add AWS ECR intel module.

### PR DESCRIPTION
**Pull request by ecdavis**
TODO: tests ;) -- please wait until I add them before reviewing

### CODE REVIEW

File: cartography/intel/aws/ecr.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import logging
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def get_ecr_repositories(boto3_session, region):
```
Comment (marco-lancini) [inline cartography/intel/aws/ecr.py:8]: would it be possible to add the `@aws_handle_regions` decorator?
Sample: https://github.com/lyft/cartography/blob/master/cartography/intel/aws/eks.py

File: cartography/intel/aws/ecr.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    client = boto3_session.client('ecr', region_name=region)
+    paginator = client.get_paginator('describe_repositories')
+    ecr_repositories = []
+    for page in paginator.paginate():
+        ecr_repositories.extend(page['repositories'])
+    return {'repositories': ecr_repositories}
+
+
+def get_ecr_repository_images(boto3_session, region, repository_name):
```
Comment (marco-lancini) [inline cartography/intel/aws/ecr.py:17]: would it be possible to add the `@aws_handle_regions` decorator?
Sample: https://github.com/lyft/cartography/blob/master/cartography/intel/aws/eks.py

File: cartography/intel/aws/ecr.py (hunk 1 above)
Comment (achantavy) [reply cartography/intel/aws/ecr.py:8]: Oh yeah, good idea.

**Comment by achantavy** [comment]
Replaced by #429

**Pull request closed without merging**